.github/workflows/ci.yml

```diff
@@ -8,6 +8,11 @@ on:
       - 'maintainers/**'
   workflow_dispatch:
 
+# Checks if any concurrent jobs under the same pull request or branch are being executed
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Only cancel this workflow.
Other workflows are super small and we don't need to kill them.